systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/PodUtils.java

```diff
@@ -234,28 +232,37 @@ public static void waitUntilPodLabelsDeletion(String podName, String... labelKey
     }
 
     /**
-     * Method waitUntilPodsByNameStability ensuring for every pod listed for kafka or zookeeper statefulSet will be controlling
-     * their status in Running phase. If the pod will be running for selected time #Constants.GLOBAL_RECONCILIATION_COUNT
-     * pod is considered as a stable. Otherwise this procedure will be repeat.
-     * @param podNamePrefix all pods that matched the prefix will be verified
+     * Method waitForPendingPod ensures that at least one pod from listed (by prefix) will be in Pending phase
```

Comment:
You don't need to include the method name in the Javadoc. 

```suggestion
     * Ensures that at least one pod from listed (by prefix) is in Pending phase
```